tests/DropDown.test.jsx

```diff
@@ -2,95 +2,193 @@ import React from 'react';
+      </DropDownItem>
+      <DropDownItem>
+        <a href="/">Cooking</a>
+      </DropDownItem>
+      <DropDownItem>
+        <a href="/">Decoration</a>
+      </DropDownItem>
+      <DropDownItem>
+        <a href="/">Home Electronics</a>
+      </DropDownItem>
+    </DropDown>
+  ));
 
   const tree = component.toJSON();
   expect(tree).toMatchSnapshot();
 });
 
-test('Renders correct default selection', () => {
-  const component = renderer.create(<DropDown
-    items={[
-        {
-          active: true,
-          icon: <img alt="Austrian flag" src="austrianflag.jpg" />,
-          link: <a href="/at" alt="hi">Austria</a>,
-          value: 'Austria'
-        },
-        {
-          active: false,
-          icon: <img alt="German flag" src="germanflag.jpg" />,
-          link: <a href="/de">German</a>,
-          value: 'German'
-        }
-      ]} />);
+test('Renders with pullRight and showOnHover false', () => {
+  const component = renderer.create((
+    <DropDown
+      pullRight
+      showOnHover={false}
+      title={(
+        <span>Österreich <span aria-label="at" role="img">🇦🇹</span></span>
+      )}>
+      <DropDownItem
+        icon={<span aria-label="at" role="img">🇦🇹</span>}>
+        <a href="/">Österreich</a>
+      </DropDownItem>
+      <DropDownItem
+        icon={<span aria-label="de" role="img">🇩🇪</span>}>
+        <a href="/de">Deutschland</a>
+      </DropDownItem>
+      <DropDownItem
+        icon={<span aria-label="ch" role="img">🇨🇭</span>}>
+        <a href="/ch">Schweiz</a>
+      </DropDownItem>
+      <DropDownItem
+        icon={<span aria-label="us" role="img">🇺🇸</span>}>
+        <a href="/us">United States</a>
+      </DropDownItem>
+    </DropDown>
+  ));
+
+  const tree = component.toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+test('Renders with direction up', () => {
+  const component = renderer.create((
+    <DropDown
+      direction="up"
+      title={(
+        <span>Österreich <span aria-label="at" role="img">🇦🇹</span></span>
+      )}>
+      <DropDownItem
+        icon={<span aria-label="us" role="img">🇺🇸</span>}>
+        <a href="/us">United States</a>
+      </DropDownItem>
+    </DropDown>
+  ));
 
   const tree = component.toJSON();
   expect(tree).toMatchSnapshot();
 });
 
 test('Renders menu when button is clicked', () => {
-  const component = mount(<DropDown
-    items={items} />);
-  component.find('button').simulate('click');
+  const component = mount((
+    <DropDown
+      showOnHover={false}
+      title={(
+        <span>United States <span aria-label="us" role="img">🇺🇸</span></span>
+      )}>
+      <DropDownItem
+        icon={<span aria-label="at" role="img">🇦🇹</span>}>
+        <a href="/">Österreich</a>
+      </DropDownItem>
+    </DropDown>
+  ));
+
+  component.find('ul button').simulate('click');
+
+  const tree = toJson(component);
+  expect(tree).toMatchSnapshot();
+});
+
+test.skip('Closes menu when click outside', () => {
```

Comment:
Still couldn't figure this out :'( Tried to fix and Googled it again, but couldn't find any way to simulate outside clicks here...